src/test/java/lab5/CircePlayerTest.java

```diff
@@ -0,0 +1,32 @@
+package lab5;
+
+import lab5.players.CircePlayer;
+import org.junit.jupiter.api.Test;
+import lab5.game.Board;
+import lab5.game.Position;
+import lab5.game.Row;
+import lab5.game.Col;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CircePlayerTest {
+    @Test
+    public void testCirceCenterFirst() {
+        CircePlayer circe = new CircePlayer("Circe");
+        Board board = new Board();
+
+        Position move = circe.pickNextMove(board);
+        Position expected = new Position(Row.Middle, Col.Middle);
+        assertEquals(expected, move);
+    }
+    @Test
+    public void testCirceSkipsCenter() {
+        CircePlayer circe = new CircePlayer("Circe");
+        Board board = new Board();
+
+        board.placeNextToken(new Position(Row.Middle, Col.Middle));    // Take the center
+        Position expected = new Position(Row.Top, Col.Middle);
+        Position move = circe.pickNextMove(board);
+        assertEquals(expected, move);
+    }
```

Comment:
Would be best to just set up a loop using a list like the `priorityOrder` you have in the actual CircePlayer class and then loop through and run a test for each, where you fill everything up to a certain point, then test that Circe picks the next.